sdk/docker/dockerSkeleton/README.md

```diff
@@ -0,0 +1,9 @@
+Blackbox Actions
+================
+
+1. Download and install the Whisk CLI by following https://github.com/openwhisk/openwhisk/blob/master/docs/README.md
+2. Use "cli download XXX" to download the Blackbox Action skeleton.
```

Comment:
This readme also seems outdated. Guess no one touched blackbox sdk in a while.
```
wsk sdk install docker
```
?